Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs

```diff
@@ -87,6 +89,20 @@ public virtual void SetupLayer()
 				_actualView.Layer.BackgroundColor = Element.BackgroundColor.ToCGColor();
 			}
 
+			_actualView.Layer.RemoveBackgroundLayer();
```

Comment:
Is this safe if the user doesn't set the Background at all? Is it basically a noop?

Reply:
The `RemoveBackgroundLayer` method internally check if there is a specific Layer, otherwise it will do nothing.